tap_zendesk/client.py

```diff
@@ -93,12 +91,9 @@ def get_url_params(
         Returns:
             A dictionary of URL query parameters.
         """
-        params: dict = {}
+        params: dict = {"page[size]": 100, "sort_order": "asc"}  # Include sort_order
```

Comment:
Something we can check is whether the maximum page size is different for each endpoint

Reply:
Thanks! Yes it is. I added support for different pagination sizes in [ef9a87b](https://github.com/Automattic/tap-zendesk-singer-sdk/pull/1/commits/ef9a87b336168ce39f1378234d50d245a1114807).